examples/PhaseCorrelationImageRegistration.cxx

```diff
@@ -72,12 +72,12 @@ int main( int argc, char *argv[] )
     }
 
   // init the registration method
-  using RegistrationType = itk::PhaseCorrelationImageRegistrationMethod< ImageType, ImageType >;
+  using RegistrationType = itk::PhaseCorrelationImageRegistrationMethod< ImageType, ImageType, float >;
   RegistrationType::Pointer pcmRegistration = RegistrationType::New();
   pcmRegistration->SetFixedImage( fixedReader->GetOutput() );
   pcmRegistration->SetMovingImage( movingReader->GetOutput() );
 
-  using OperatorType = itk::PhaseCorrelationOperator< itk::NumericTraits< PixelType >::RealType, Dimension >;

```

Comment:
Why force `float`?

Reply:
Results are pretty much the same, but float is some 10% or so faster.